lexicon.txt

```diff
@@ -0,0 +1,1317 @@
+networkbufferdescriptor
+networkinterface
+nextidx
+nic
+nobroadcast
+noevent
+nondet
+noninfringement
+nop
+ntoh
+nulling
+nvic
+nxt
+obcr
+oe
+ok
+onconnect
+onconnected
+onreceive
+onsent
+op
+opendns
+optimisation
+optimise
+optimised
+optlen
+ored
+org
+osf
+outfile
+pacap
+pace
+param
+parsednsreply
+passcontrolframes
+pauselowthreshold
+pausetime
+pbl
+pbuf
+pbufs
+pc
+pcallback
+pcallbackfunction
+pcap
+pcapplicationhostnamehook
+pcbuffer
+pce
+pcf
+pcfrom
+pchostname
+pcmessage
+pcname
+pcnetworkbuffer
+pcs
+pcto
+pd
+pdata
+pdc
+pdecode
+pdelay
+pdelayreq
+pdelayresp
+pdfail
+pdfalse
+pdpass
+pdtrue
+perfectfilter
+perfecthashtablefilter
+pfc
+pfnz
+phy
+phyaddress
+phyhandling
+phyreg
+phyrr
+phys
+pio
+pioa
+pll
+plposition
+plt
+pm
+pma
+pmacobject
+pmc
+pme
+pmecr
+pmt
+posix
+pparam
+ppkt
+ppointer
+pr
+pre
+printf
+printfs
+processdhcpreplies
+processippacket
+projdefs
+promiscuousmode
+prvallowippacket
+prvcachelookup
+prvchecknetworktimers
+prvcheckoptions
+prvcheckrxdata
+prvcopydatatoqueue
+prvcreatednsmessage
+prvdeterminesocketsize
+prvemacdeferredinterrupthandlertask
+prvemachandlertask
+prvgethostbyname
+prvhandleestablished
+prvhandlelisten
+prvhandlesynreceived
+prvinitialisedhcp
+prvlinkstatuschange
+prvnotifyqueuesetcontainer
+prvparsednsreply
+prvprocessdhcpreplies
+prvprocessdnscache
+prvprocessippacket
+prvprocessnetworkdownevent
+prvreadnamefield
+prvreadsackoption
+prvrxcallback
+prvsenddata
+prvsetoptions
+prvsetsynackoptions
+prvsinglesteptcpheaderoptions
+prvskipnamefield
+prvstorerxdata
+prvtcphandlefin
+prvtcphandlestate
+prvtcpprepareconnect
+prvtcppreparesend
+prvtcpreturnpacket
+prvtcpsendpacket
+prvtcpsendrepeated
+prvtcpsendreset
+prvtcpsocketcopy
+prvtcpsocketisactive
+prvtcpstatusagecheck
+prvtxcallback
+prvvalidsocket
+prvwinpcaprecvthread
+prvwinpcapsendthread
+prvwritepackets
+ps
+pt
+pte
+pthread
+pthreads
+ptp
+ptxpkt
+ptz
+pucbuffer
+pucbyte
+pucdata
+pucethernetbuffer
+pucgetuncachedmemory
+pucinterfacename
+pucname
+pucnewbuffer
+pucnextdata
+pucptr
+pucrecvdata
+pucstart
+pucudppayload
+pucudppayloadbuffer
+pucusebuffer
+puldelay
+pulipaddress
+pullups
+pulnumber
+pulvalue
+pvbuffer
+pvdata
+pvoptionvalue
+pvparam
+pvparameters
+pvportmalloc
+pvportmallocsocket
+pvsearchid
+pxackmessage
+pxaddr
+pxaddress
+pxallnetworkinterfaces
+pxanswer
+pxbuffer
+pxbufferdescriptor
+pxdescriptor
+pxdnsmessageheader
+pxduplicatenetworkbufferwithdescriptor
+pxgetnetworkbufferwithdescriptor
+pxhandleconnected
+pxhandlereceive
+pxhead
+pxheadsegment
+pxipheader
+pxiterator
+pxlength
+pxlist
+pxlistfindlistitemwithvalue
+pxmacaddress
+pxmessage
+pxnetworkbuffer
+pxnextbuffer
+pxnextnetworkbufferdescriptor
+pxontcpconnected
+pxontcpreceive
+pxontcpsent
+pxonudpreceive
+pxonudpsent
+pxopenedinterfacehandle
+pxpeersocket
+pxphyobject
+pxprops
+pxprotocolheaders
+pxprotpack
+pxsegment
+pxsocket
+pxsocketset
+pxtcpheader
+pxtcppacket
+pxtcpsocketlookup
+pxtcpwindow
+pxudpsocketlookup
+pxwindow
+qmu
+que
```

Comment:
this looks like a misspelling of _queue_.

Reply:
It was a misspelling of queue. I have updated lexicon.txt to reflect that.